weave/trace_server/trace_server_interface.py

```diff
@@ -692,6 +692,26 @@ class FeedbackPurgeRes(BaseModel):
     pass
 
 
+class ActionsReadStaleReq(BaseModel):
+    pass
+
+
+# TODO: Merge this with schema in ActionsExecuteBatchReq
+class ActionSchema(BaseModel):
```

Comment:
these classes don't belong here, they belong closer to where they are used as they are describing the implementation schema, not the external interface